doc/cnn_1D_segm.txt

```diff
@@ -0,0 +1,201 @@
+.. _cnn_1D_segm:
+
+Network for 1D segmentation
+***************************
+
+.. note::
+    This section assumes the reader has already read through :doc:`lenet` for 
+    convolutional networks motivation and :doc:`fcn_2D_segm` for segmentation
+    standard network.
+
+
+Summary
++++++++
+
+The fundamental notions behind segmentation have been explained in :doc:`fcn_2D_segm`.
+A particularity here is that some of these notions will be applied to 1D
+segmentation. However, almost every Lasagne layer used for 2D segmentation have
+their respective 1D layer, so the implementation would look alike if the same
+model was used.
+
+
+
+
+Data
+++++
+
+The `BigBrain <https://bigbrain.loris.ca/main.php>`__ dataset is a 3D ultra-high resolution model of the brain reconstructed from 2D sections.
+We are interested in the outer part of the brain, the cortex.
+More precisely, we are interested in segmenting the 6 different layers of the cortex in 3D.
+Creating an expertly labelled training dataset with each 2D section (shown in figure 1) is unfeasible. Instead of giving as input a 2D image of one section of the brain, we give as input 1D vectors with information from across the cortex, extracted from smaller portions of manually labelled cortex
+as shown in Figure 2. The actual dataset can be found `here TODO: link the dataset`.
```

Comment:
Ah other one. I am waiting for the link from someone. The first link is where the actual dataset came from, but our actual dataset is preprocessed in some way from BigBrain